### DISCUSSION TITLE
TTS stopping while reading

### DISCUSSION THREAD

**Discussion by wenrolland** [n4ze3m/page-assist#199] [General]
Hi! I really love the app! The only thing is that when I try using the TTS to read a section, it just stops after a couple of seconds. I'm using the latest version (1.2.3) and Google Voice. Anyone having this problem ?

**Comment by n4ze3m** [comment]
Hey, thanks for reporting it! I will fix it in the upcoming version :)

It is because of this: https://stackoverflow.com/questions/21947730/chrome-speech-synthesis-with-longer-texts. I didn’t know about this issue with TTS :)
